torch/_torch_docs.py

```diff
@@ -1465,6 +1465,47 @@ def merge_dicts(*dicts):
     tensor([ 0.5000, -0.4702, -0.4599,  0.5000])
 """.format(**common_args))
 
+add_docstr(torch.complex,
+           r"""
+complex(real, imag, out=None) -> Tensor
+
+Constructs a complex-valued tensor from Cartesian coordinates.
+
+:attr:`real` and :attr:`imag` should be floating-point tensors of the same
+dtype. ``torch.float32`` tensors will lead to an output of ``torch.complex64``,
+and ``torch.float64`` tensors will lead to an output of ``torch.complex128``.
+
```

Comment:
Add "Args:" and "Keyword args:" (for out) here. See what `torch.add` does as a reference.